darwin/cli_functions.py

```diff
@@ -719,3 +729,12 @@ def _load_client(
 def _console_theme():
     return Theme({"success": "bold green", "warning": "bold yellow", "error": "bold red"})
 
+
+def print_new_version_info(client):
+    if client and not client.newer_darwin_version():
+        return
+    (a, b, c) = client.newer_darwin_version()
+    print(
+        f"A newer version of darwin-py is available [{a}.{b}.{c}]\ninstall with: pip install darwin-py=={a}.{b}.{c}",
+        file=sys.stderr,
+    )
```

Comment:
Why don't we use `rich` to print a `warning` message here?
Also, why are we writing into `sys.stderr`?